code/modules/reagents/reagent_containers/food/snacks/grown.dm

```diff
@@ -775,3 +775,68 @@
 			reagents.add_reagent("toxin", 1 + round(potency/10, 1))
 			reagents.add_reagent("spiritbreaker", 10)
 			bitesize = 1+round(reagents.total_volume/2, 1)*/
+
+/obj/item/weapon/reagent_containers/food/snacks/grown/nofruit
+	name = "no-fruit"
+	desc = "Any plant you want, at your fingertips."
+	icon_state = "nofruit"
+	potency = 15
+	filling_color = "#FFFCCC"
+	plantname = "nofruit"
+	var/list/available_fruits = list()
+	var/switching = 0
+	var/current_path = null
+	var/counter = 1
+
+/obj/item/weapon/reagent_containers/food/snacks/grown/nofruit/New()
+	..()
+	available_fruits = typesof(/obj/item/weapon/reagent_containers/food/snacks/grown)
```

Comment:
What criteria does existing_typesof() use? I might use it on the Roulette Revolver as well.